snorkel/classification/training/loggers/log_writer.py

```diff
@@ -71,7 +85,7 @@ def write_config(
         config_filename
             Name of file in logging directory to write to
         """
-        self.write_json(config, config_filename)
+        self.write_json(config._asdict(), config_filename)
```

Comment:
does `asdict()` still count as a private function?

Reply:
Nope! It's documented: https://docs.python.org/3/library/collections.html#collections.somenamedtuple._asdict

Underscore is to prevent collision with a field name